pyccel/stdlib/math/pyc_math.f90

```diff
@@ -0,0 +1,149 @@
+module pyc_math
+
+implicit none
+
+real(kind=8), parameter, private :: pi = 4.D0 * DATAN(1.D0)
```

Comment:
I think we should use ISO_C_BINDING to define the kind type parameter, as we do in the Fortran printer now.

@saidctb @EmilyBourne Do you agree?

Reply:
Yes I agreee

Reply:
yes I agree